package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "js-marketing-notification",
-  "version": "1.2.1",
+  "version": "1.3.1",
   "description": "A module for displaying tasteful marketing content to users upon first visit to page.",
-  "main": "src/js-marketing-notification.js",
+  "main": "js-marketing-notification.ts",
```

Comment:
Never mind looks like we were including that already before. So what does this change handle exactly?